cypress/fixtures/users.json

```diff
@@ -0,0 +1,126 @@
+{
+    "page": 1,
+    "totalPages": 1,
+    "users": [
```

Comment:
As much as I enjoy the test data here 🙃 it may be a hassle to update this when we start passing more fields back to the client (i.e. all the other fields on `User`) - can we instead generate this using some of the test utils that we've created for the server?